source/includes/fundamentals/code-snippets/libmongocrypt-maven-versioned.rst

```diff
@@ -0,0 +1,10 @@
+.. code-block:: xml
+
+   <dependencies>
+       <dependency>
+           <groupId>org.mongodb</groupId>
+           <artifactId>mongodb-crypt</artifactId>
+           <version>1.1.0-beta1</version>
```

Comment:
If you're not going to publish this for a while, you can just change to to 1.1.0